CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+cmake_minimum_required(VERSION 2.8)
+
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/generic-msp430-gcc.cmake")
+
+# Name your project
+project(EXAMPLE)
+
+#device dependent config, here for launchpad
+set(MSP430_MCU msp430fr2355)
+set(MSP430_MCU_FREQ 24000000UL)
+
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD_REQUIRED ON)
+# set(CMAKE_C_EXTENSIONS ON)
```

Comment:
Any reason this is here commented out?

Reply:
I'll remove it. It was just to see if we were using any extensions.